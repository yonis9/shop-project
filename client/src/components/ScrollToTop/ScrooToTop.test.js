import React from 'react';
import { shallow } from 'enzyme';

import { ScrollToTop } from './ScrollToTop';

describe('ScrollToTop component', () => {
    let wrapper;
    let mockHistory;

    beforeEach(() => {
        mockHistory = {
            listen: jest.fn()
        }

        wrapper = shallow(<ScrollToTop history={mockHistory}/>);
    });

    it('should render ScrollToTop component', () => {
        expect(wrapper).toMatchSnapshot();
    });
});