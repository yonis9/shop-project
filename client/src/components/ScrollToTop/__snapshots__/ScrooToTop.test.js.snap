// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop component should render ScrollToTop component 1`] = `ShallowWrapper {}`;
